developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -871,16 +969,28 @@ KMX_DWORD ProcessSystemStore(PFILE_KEYBOARD fk, KMX_DWORD SystemID, PFILE_STORE
   case TSS_COPYRIGHT:
     break;
 
+  case TSS_CUSTOMKEYMANEDITION:
+    break;
+
+  case TSS_CUSTOMKEYMANEDITIONNAME:
+    break;
```

Comment:
Added for completeness along with TSS_BASELAYOUT above and several others below -- now all TSS_ values are checked here. This change is necessary because previously we didn't check the return value from `AddStore()` in several calls to `CompileKeyboardBuffer()`, which was poor form.